cmd/gdeploy/commands/identity/sso/enable.go

```diff
@@ -215,14 +215,19 @@ func updateOrAddSSO(c *cli.Context, idpType string) error {
 		// sort groupNames alphabetically
 		sort.Strings(groupNames)
 
-		err = survey.AskOne(&survey.Select{
-			Message: "The ID of the Granted Administrators group in your identity provider:",
-			Options: groupNames,
-		}, &chosenKey)
+		if len(groupNames) == 0 {
+			clio.Warn("no groups found please make at least 1 group in your identity provider")
```

Comment:
This should return an error rather than a warning otherwise the administratorID will be empty and deployment will fail.

The user should be given a more useful error message and potentially fall back to manual group ID input field.
This will allow the user to create the group then continue the sso setup without losing the values they already added.